### ISSUE TITLE
usage with cluster 

### ISSUE THREAD

**Issue by vectorselector** [dtsepelev/meteor-up-letsencrypt#3]
I have an issue with mupx-letsencrypt@meteor14 in which I'm using 2 appservers and meteorhacks "cluster"

`   
 {
      "host": "1.2.3.4",
      "username": "myuser",
      "pem": "/Users/me/.ssh/id_rsa",
      "env": {
           "CLUSTER_BALANCER_URL": "https://app2.whatever.com"}
    }
`
the CLUSTER_BALANCER_URL doesn't seem to resolve, and I've added the appserver dns names to the letsEncrypt object, and i'm not sure how this can inform the "servers" object...

`
  "letsEncrypt": {
    "domain": "domain.com,www.domain.com,app1.domain.com,app2.domain.com",
    "email": "me@domain.com"
  }
`

basically, the problem is that when I define a  "CLUSTER_BALANCER_URL"
my spinner just spins (iron-router loading) and the meteor app presumably doesn't connect it's cluster to any cluster load balancers...

any ideas?

thank you, by the way, for the only integrated mup ssl letsencrypt solution to date, even after over 2 years of blah blah online from loads of people... hats off respect

**Comment by dtsepelev** [comment]
@vectorselector unfortunately I didn't have any experience with meteorhacks cluster, but it looks like let's encrypt container should be installed behind the balancer. Maybe someone with cluster experience could share more details, cause I do not have any ideas how to fix it without digging into cluster workflow.

**Comment by vectorselector** [comment]
So, I had some success putting the letsencrypt object inside of the server object in parallel with it's pem and env properties, as well as specifying the specific endpoint url (i used the IP address after unknown issues with serverhost.mydomain.com)

      "host": "1.2.3.4",
      "username": "myuser",
      "pem": "/Users/me/.ssh/id_rsa",
	  "letsEncrypt": {
		"domain": "mydomain.com,www.mydomain.com",
		"email": "me@mydomain.com"
	  },
      "env": {
           "CLUSTER_ENDPOINT_URL": "https://1.2.3.4",
           "CLUSTER_BALANCER_URL": "https://1.2.3.4"
      }
    

this would confirm what you suggest but I still wonder if cluster is really balancing with it's balancing urls...the SSL part is what confuses me, as DNS entries for serverhost.mydomain.com no longer resolve... anyway, sorta closed. let's close it unless you wish to add info...
